easybuild/easyconfigs/b/BWA/BWA-0.7.17-foss-2023b.eb

```
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA, HPCNow!
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+#             Jordi Blasco <jordi.blasco@hpcnow.com>, Guillermo Piccoli <guillermo.piccoli@hpcnow.com>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+#
+# Version >= 0.7.17
+# Author: Adam Huffman, Big Data Institute, University of Oxford
+#
+# Note that upstream development is mainly at: https://github.com/lh3/bwa
+##
+name = 'BWA'
+version = '0.7.17'
+homepage = 'http://bio-bwa.sourceforge.net/'
```

```suggestion
homepage = 'https://bio-bwa.sourceforge.net/'
```
Looks like `https` works now, at last.